readme.txt

```diff
@@ -1,6 +1,6 @@
 === Twenty Twenty-Four ===
 Contributors: wordpressdotorg
-Requires at least: 6.4
+Requires at least: 6.2
```

Comment:
Why is this down to 6.2? All previous default themes had the requires at least to the version it was shipped with.

Reply:
I agree, but this is just what's on core right now. We can fix that with https://github.com/WordPress/twentytwentyfour/pull/516